docs/aerodromes/classd/Archerfield.md

```diff
@@ -69,6 +69,64 @@ The circuit direction changes depending on time of day and runway being used.
 
 Circuits are to be flown at `A010`.
 
+## Helicopter Operations
+Archerfield has three helipads located around the aerodrome:
+
+- **Central Helipad**, on the southern side of Taxiway Foxtrot
+- **Northern Helipad**, adjacent the north-eastern end of Runway 04R/22L
+- The **Queensland Government Air facility** used by rescue helicopters, north of Taxiway B5
+
+!!! note
+    Both the **Northern** and **Southern** helipads are inside the maneuvering area and treated like a runway, requiring a takeoff and landing clearance from AF ADC. Helicopters operating on other surfaces outside the maneuvering areas should be instructed to *"report airborne"*  or *"report on the ground"* so the controllers knows when they are active / no longer active in their airspace. 
+
+### Training Areas
+There are 3 designated helicopter training areas at Archerfield:
+
+**Area A**: North of Taxiway B within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area B**: North and west of Runway 04L within the confines of the aerodrome boundary, clear of all buildings, not above `A005`. Circuit direction and frequency as per ATIS. A maximum of 3 helicopters are permitted to operate circuits concurrently. Additionally, a maximum of 2 helicopters are permitted to operate in ground effect only.
+
+**Area C**: South and east of Taxiway D, inside the fixed wing circuit for runway 04R/22L, east of the control tower, not above `A008`. Landings must be conducted north of Taxiway B. Once established, the only radio communications are an airborne call before each circuit and a downwind call by pilots. A maximum of 2 helicoptesr are permitted to operate in this area concurrently.
+
+<figure markdown> 
+  ![Archerfield Helicopter Training Areas](img/ybaf_helicopter_area.png){ width="450" }
+  <figcaption>Archerfield Helicopter Training Areas</figcaption>
+</figure>
+
+
+### Departures
+Helicopters comply with fixed wing outbound procedures. Departures shall be parallel to and clear of the duty runways. Departures with intended entry into BN CTR shall advise AF SMC on taxi with "Uni Departure". No heads-up coordination with BN TCU is required.
```

Comment:
What is the 'Uni Departure'?

Reply:
"Uni Departure: is a departure to the north to via UQLD, the University of Queensland. Where approaching UQLD aircraft contact Brisbane Center for clearance to enter CTA / complete airwork or orbits around the city. I wrote it the way the ERSA has it, but it's mainly targeted towards helicopters with *intended entry into the BN CTR northbound*.

Reply:
Ok no worries, even if we can just explain what a Uni Departure is. We want controllers to be able to reference the SOPs and nothing else, and get a good understanding of how to process different aircraft despite having no local knowledge. Is it limited to helicopters, or does it apply to fixed wing as well?

Reply:
Limited to helicopters, fixed wing aircraft departing the north depart track via Walter Taylor Bridge.

Reply:
Updated Archerfield.md to explain to controllers what the Uni Departure is with a bit of context around it, also added a phraseology example to Brisbane.md TCU SOPs for the controllers to understand how an airways clearance would be issued.